provisioning/roles/openldap/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+- name: Preseed slapd configuration options through debconf
+  debconf: name=slapd question={{ item.question }} value={{ item.value }} vtype={{ item.type }}
+  with_items:
+    - { question: slapd/internal/adminpw, value: admin, type: password }
```

Comment:
I think you should either make the password configurable, or add the credentials to the documentation. Also what's the difference between "adminpw" and "password1"?